base_layer/core/src/transactions/transaction_protocol/proto/transaction_sender.proto

```diff
@@ -37,9 +37,6 @@ message TransactionSenderMessage {
     oneof message {
         bool None = 1;
         SingleRoundSenderData single = 2;
-
-        // TODO: Three round types
```

Comment:
When we need multi party transaction negotiation we will add it, TODO seems redundant.